jsonschema-core.md

```diff
@@ -895,49 +895,44 @@ by other parties.
 ### Base IRI, Anchors, and Dereferencing
 
 To differentiate between schemas in a vast ecosystem, schemas are identified by
-[IRI](#rfc3987), and can embed references to other schemas by specifying their
-IRI.
+[absolute IRIs](#rfc3987) (without fragments) and can embed references to other
+schemas by specifying their respective IRIs. When comparing IRIs for the
```

Comment:
This sentence doesn't sound right. I think there's something missing?

```suggestion
[absolute IRIs](#rfc3987) (without fragments) and can be used to embed references to other
schemas by specifying their respective IRIs. When comparing IRIs for the
```

Reply:
I think "which can be used to" is probably what's intended.  The IRIs are doing the embedding, not the schemas.  Then the "by specifying their respective IRIs" probably just gets removed.

Reply:
Yes, that sounds better.

Does the "embed" part make sense? Should it just say "to reference schemas".

Reply:
What about this?

> To differentiate between schemas in a vast ecosystem, schemas are identified by [absolute IRIs](#rfc3987) (without fragments).  These identifiers are used to create references between schemas.  When comparing...